### ISSUE TITLE
Having issues with Azure AD and beta 303

### ISSUE THREAD

**Issue by JoeRedDogg** [Requarks/wiki#1050]
**Question**
I am getting an invalid username/password error after I successfully logging into Azure AD. 
After entering my password, it goes back to Wiki.js and throws the error that you see in the attached image. Is there anything that I can look at in order to troubleshoot this a little better?
I have followed the directions for Azure AD integration.

**Host Info (please complete the following information):**
OS: [Ubuntu 18.04]
Wiki.js version: [2.0.0-beta.303]
Database engine: [postgres 11]

![image](https://user-images.githubusercontent.com/55552326/65283870-1f202d80-db06-11e9-8abb-d100411ed0f2.png)

**Comment by JoeRedDogg** [comment]
Any info that could help?

**Comment by D3mon86** [comment]
Hi Check if you have filled email address for the account in AD. By default as I understand it tryies to find email that connected to azure exchange

**Comment by NGPixel** [comment]
You must have an email address set to the email field in your Azure AD. Specify the field name in the settings if you're using a custom one.

**Comment by ginomessmer** [comment]
@NGPixel Just installed v2 RC and I'm loving it so far, however, I encountered the same issue. 

Upon research and checking with MS Graph Explorer, it happens for users who don't have an Exchange Online license associated (reference https://stackoverflow.com/a/51185810). We have set alternative email addresses in Azure AD but they won't be picked up by Wiki.js. Any chance to alter this behavior in a future release? That would be awesome.
